docs/useNotify.md

```diff
@@ -216,4 +221,47 @@ export const ConnectionWatcher = () => {
 };
 ```
 
-Note that if you use this ability to pass a React node, the message will not be translated - you'll have to translate it yourself using [`useTranslate`](./useTranslate.md).
\ No newline at end of file
+Note that if you use this ability to pass a React node, the message will not be translated - you'll have to translate it yourself using [`useTranslate`](./useTranslate.md).
+
+If you have custom actions in your notification element, you can leverage the `useCloseNotification` hook to close the notification:
+
```

Comment:
I think this example is bringing confusion. Earlier in the doc, we mention 'undoable' notifications (without explaining how to handle the undo). Here, you introduce a "cancellable" notificaiton, which seems like it's the same thing, but it uses a different syntax...

So either you should find another example for this feature, or we should document how to register an undoable action.

Reply:
Any suggestion for another example?